lock/etcd.go

```diff
@@ -25,19 +27,29 @@ import (
 
 const (
 	keyPrefix       = "coreos.com/updateengine/rebootlock"
-	holdersPrefix   = keyPrefix + "/holders"
-	SemaphorePrefix = keyPrefix + "/semaphore"
+	groupBranch     = "groups"
+	semaphoreBranch = "semaphore"
+	SemaphorePrefix = keyPrefix + "/" + semaphoreBranch
```

Comment:
on actually thinking about this SemaphorePrefix is just a bad name right? what about defaultSemaphorePath?